plugin.video.otaku/resources/lib/AniListBrowser.py

```diff
@@ -3248,6 +3265,80 @@ def get_relations_res(self, variables):
         json_res = results['data']['Media']['relations']
         return json_res
 
+    def get_watch_order_res(self, variables):
+        query = '''
```

Comment:
I'm not entirely happy with getting such a large volume of unused data but sticking to what Goldenfreddy asked for. I come from a space where we build microservices that perform specific functions and because of the high volume of messages we only get the data we're actually planning to use.

Reply:
Hey, its understandable but we have always have cache so next when we load up the watch order, it will load faster. Great feature have done a really great job with this. Thank you so much for adding this feature and alot of people are going to be really happy with this.